### PR TITLE
Fixes for Swift Package Manager

### PR DESCRIPTION
**Pull request by macinspak**
Hi Gabriel,

Thanks for your patience while I tested out the changes thoroughly. It's working solidly now and apologies for missing a couple of things on the last pull request.

I've also updated the documentation to include instructions for SPM usage.

Please note, you will need to tag your release (I've nominated 0.3.4 in the README as that's your next revision number). If you don't do that, the instructions for SPM will most likely not work and you'll need to change the README to use a commit number or something in the SPM config.

Thanks again for such a useful library.

Andrew

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,12 +11,30 @@ The YAJL framework is an Objective-C framework for the [YAJL](http://lloyd.githu
...
+
+## CocoaPods
 
 ```ruby
 pod "YAJLO"
 ```
 
+## Swift Package Manager
+
+You can use The Swift Package Manager to install SwiftyJSON by adding the proper description to your Package.swift file:
```
Comment (gabriel) [inline README.md:24]: I'm confused about what this is about?

Comment (gabriel) [inline README.md]: Should this say yajl-objc?

File: README.md [hunk 2]
```diff
@@ -11,12 +11,30 @@ The YAJL framework is an Objective-C framework for the [YAJL](http://lloyd.githu
...
+
+```swift
+// swift-tools-version:5.5
+import PackageDescription
+
+let package = Package(
+    name: "YOUR_PROJECT_NAME",
```
Comment (gabriel) [inline README.md:31]: Should name be yajl-objc?

**Review by gabriel — CHANGES_REQUESTED** [review]

**Comment by gabriel** [comment]
Thanks for this!

**Comment by macinspak** [comment]
Yes, that is just stupid isn’t it? I’ll fix it. Sorry again. Not great for confidence is it? 🙃

**Comment by macinspak** [comment]
Fixed

**Pull request merged**